docs/developer/accessibility.md

```diff
@@ -0,0 +1,25 @@
+# Accessibility
+
+FAST-DNA has worked diligently from the ground up to enable accessibility in collaboration with legal, accessibility, and marketing teams at Microsoft. As an ecosystem to accelerate the building of Web sites and apps we ensure you have the tools to build accessible user exerpiences.
+
+To learn more about Microsoft's commitment visit their [accessibility](https://www.microsoft.com/en-us/accessibility) website.
+
+Compliments of [@rachelnabors](https://github.com/rachelnabors)
+
+## Browsers
+
+The base component packages support IE11+ and all modern browsers.
+
+### Reporting browser bugs
+
+* [Edge](https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/)
+* [Firefox](https://bugzilla.mozilla.org/)
+* [Safari](https://bugs.webkit.org/)
+* [Chrome](https://bugs.chromium.org/p/chromium/issues/list)
+* [Opera](https://bugs.opera.com/wizard/desktop)
```

Comment:
I don't think we need to include bug reporting links to other projects, it also has nothing specifically to do with accessibility.

Reply:
Everything in this documentation at one point was requested or a gap and that's why we have it. I have moved it into standards in PR.